tsconfig.json

```diff
@@ -0,0 +1,75 @@
+{
+    "extends": "@tsconfig/svelte/tsconfig.json",
+    "compilerOptions": {
+        /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+        /* Basic Options */
+        // "incremental": true,                         /* Enable incremental compilation */
+        "target": "esnext" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', 'ES2021', or 'ESNEXT'. */,
+        "module": "esnext" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */,
```

Comment:
how does this option affect code or build?

Reply:
it prevents warning messages when `npm run build`

Reply:
and the four scripts in `package.json`
- `npm run build`
- `npm run dev`
- `npm start`
- `npm run check`

are work perfectly :yum:

so, there's no issue when adit `tsconfig.json` file